docs/Theming-Fonts-And-Typography.md

```diff
@@ -0,0 +1,14 @@
+# Fonts & Typography
+
+[Back to Theming](./Theming.md)
+
+Fonts are defined in `app/theme/typography.ts`. We use a similar approach to [colors](./Theming-Colors-And-Palettes.md), defining a set of fonts and then using those fonts to define semantic font names to be used throughout the app. This allows us to have a consistent font usage across the app, and also allows us to change the fonts easily.
+
+## Fonts
+
+We define the fonts used in `app/theme/typography.ts`. The custom fonts are loaded using the `useFonts` hook from [`expo-fonts`](https://docs.expo.dev/guides/using-custom-fonts/) to load the fonts.
+
+## Typography
+
+As stated earlier, we define semantic names to be used throughout the app, similar to how we handle [colors](./Colors-And-Palettes.md. For example, we want to have consistent heading fonts across the app, so we will define a `heading` font. We can then use this font in our components. See the [Text](./../Components-Text.md) component for an example. We load the `typography.primary` font in the `Text` component, and switch between the different weights depending on what font weight is specified.
```

Comment:
>[colors](./Colors-And-Palettes.md.

missing paren

---

since we use a custom Text component that encapsulates most of the text in the app, the "semantic" names are essentially presets. Again, presets should only be created for instances where there's a style pattern that is constantly used in the app. For one-off cases, it's recommended to use the `size` and `weight` props in Text.